makecomparison.sh

```diff
@@ -31,9 +31,15 @@ echo "$git_message"
 echo "-----------"
 echo "The testresult is $testresult"
 
-# if on staging visual tests pass, unhold the production pipeline.
-if [ "$APP_ENVIRONMENT" = 'staging' ] && [ $testresult -eq 0 ] && [[ "$git_message" != *"[HOLD]"* ]]; then
-  ./promote.sh "$CIRCLE_WORKFLOW_ID"
+# Some extra checks on staging
+if [ "$APP_ENVIRONMENT" = 'staging' ]; then
+  # if visual tests pass, unhold the production pipeline
+  # else notify on Slack
+  if [ $testresult -eq 0 ] && [[ "$git_message" != *"[HOLD]"* ]]; then
+    ./promote.sh "$CIRCLE_WORKFLOW_ID"
+  else
+    ./slack_notify.sh
```

Comment:
Not sure if we have two links there :thinking: Because ideally we do want to link to the whole workflow in order to approve it, instead of going back and forth.

Reply:
> We could put the mismatch count that is used in [this check](https://github.com/greenpeace/planet4-backstop/pull/7/files#diff-701ad189d150ca1b0a314ee77097bc26R16) into a variable, and include it the notification.

Is the link correct? It points to the diff of the PR.

Reply:
If you check the `activity-ci` channel and scroll back you'll see some examples of how this looks (search for red color notifications). This is currently linking directly to the workflow in discussion, which I find more practical, to be able to have the approval job also available.